tox.ini

```diff
@@ -1,6 +1,6 @@
 # tox.ini
 [tox]
-envlist = py27
+envlist = py27, py34
```

Comment:
I guess if you want to run tests with multiple environments, you should also modify `.travis.yml` by using extending `env` like [here](https://github.com/scrapinghub/shub/blob/master/.travis.yml#L4) OR using `matrix` instead of `python:2.7` like it's done [here](https://github.com/scrapinghub/python-scrapinghub/blob/master/.travis.yml#L2). For now Travis run the [tests](https://travis-ci.org/scrapy-plugins/scrapy-pagestorage/builds/235946859) only with python 2.